src/index.html

```diff
@@ -1,17 +1,118 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
Don't add blanklines between parent and child tags, fix it everywhere:
```suggestion
<html lang="en">
  <head>
```